src/xenia/cpu/backend/x64/x64_sequences.cc

```diff
@@ -1822,6 +1969,22 @@ struct DIV_I64 : Sequence<DIV_I64, I<OPCODE_DIV, I64Op, I64Op, I64Op>> {
       assert_true(!i.src1.is_constant);
       e.mov(e.rcx, i.src2.constant());
       if (i.instr->flags & ARITHMETIC_UNSIGNED) {
+        auto div_shift_info = magicu<uint64_t>(i.src2.value->constant.u32);
+		//todo: handle addflag
```

Comment:
1. Tabs instead of spaces.
1. Don't leave this silently unhandled. Add an assert for this case where the implementation should go.